### ISSUE TITLE
NFC: ATQA on the screen is not equal to ATQA in dump file

### ISSUE THREAD

**Issue by Mixaill** [flipperdevices/flipperzero-firmware#2067]
### Describe the bug.

ATQA on the flipper screen mismatch with the ATQA inside the dump file (bytes are swapped)
@gornekich 

1) flipper screen
![Screenshot-20221130-231551](https://user-images.githubusercontent.com/704382/204899282-db0f416c-9d0c-4d6d-8d48-d4b726a7637d.png)

2) dump file
```
...
# UID, ATQA and SAK are common for all formats
UID: E9 1C F5 3C
ATQA: 04 00
SAK: 08
# Mifare Classic specific data
Mifare Classic type: 1K
Data format version: 2
# Mifare Classic blocks, '??' means unknown data
Block 0: E9 1C F5 3C 3C 08 04 00 62 63 64 65 66 67 68 69
```

### Reproduction

1. read NFC tag
2. save it to the file
3. open dump info on the flipper zero screen
4. open dump file on the PC
5. compare ATQA

### Target

0.72.0-rc

**Comment by gornekich** [comment]
Hello @Mixaill 
Thanks for bug report. Will fix it soon

**Comment by Astrrra** [comment]
Fixed in #2098, please create a new issue if the problem persists.
